### PR TITLE
FEATURE: Add Behat tests suite

### PR DESCRIPTION
**Pull request by dfeyer**

### CODE REVIEW

**Comment by dfeyer** [comment]
Work in Progress

- [x] Adapt to Neos and master branch of the package
- [ ] Polish the test a bit
- [ ] Merge #11 if needed, if not adapt the test

@gerhard-boden are you fine with merging this only on master ? We can setup travis when it's done. 

I have want to work on some features, and will be more confortable to have a solid test suite before improving the package.

**Comment by gerhard-boden** [comment]
It's really awkward for me, since I haven't gotten around to take a good close look at this (might be done in minutes if everything works) 🙈  

The thing is: After two years this topic popped up (as happy as I am it did) in the most stressful time and I'm currently working graveyard shifts.. And zero time on the weekend as well, since thins pile up during the week which need to be taken care of then (including social stuff) 🙄

TLDR; I'll look at it as soon as possible 😃

**Comment by gerhard-boden** [comment]
image

Since we merged the bugfix https://github.com/neos/redirecthandler-neosadapter/pull/11 and everything is 💚 after the rebase I think we're ready to merge this now. 

@dfeyer @kdambekalns please confirm, afterwards we can add these tests to the Travis CI.
